exercises/reverse-string/example.jl

```diff
@@ -0,0 +1,12 @@
+function reversestring(phrase::String)
```

Comment:
The type shouldn't be part of the name, it's already clear from the signature. Since `reverse` is already defined in Julia Base, perhaps `myreverse` or `reverse_` would be better names.